crates/trie/common/benches/prefix_set.rs

```diff
@@ -115,8 +111,7 @@ fn prefix_set_bench<T>(
 fn generate_test_data(size: usize) -> (Vec<Nibbles>, Vec<Nibbles>, Vec<bool>) {
     use prop::collection::vec;
 
-    let config = ProptestConfig { result_cache: basic_result_cache, ..Default::default() };
-    let mut runner = TestRunner::new(config);
+    let mut runner = TestRunner::deterministic();
```

Comment:
I'm not sure what impact the result cache originally had on benchmark performance here, but can we keep it? See the proptest docs for what the deterministic function desugars to:
https://docs.rs/proptest/latest/proptest/test_runner/struct.TestRunner.html#method.deterministic

Reply:
thank you, i just refactor the code, please help me review